core/tests/puppeteer/lighthouse_setup.js

```diff
@@ -505,6 +518,15 @@ const main = async function () {
   await getSkillEditorUrl(browser, page);
   await generateDataForTopicAndStoryPlayer(browser, page);
   await generateDataForClassroom(browser, page);
+
+  fs.writeFileSync(
+    'core/tests/puppeteer/.env',
+    `exploration_id=${explorationId}\n` +
+      `story_id=${storyId}\n` +
+      `topic_id=${topicId}\n` +
+      `skill_id=${skillId}\n`
+  );
```

Comment:
I think it's fine to transfer data this way instead of through the process's stdout like we used to do, but let's only use one method. Can you remove the code that this `.env` file makes redundant? For example: https://github.com/oppia/oppia/blob/03fc029568cc0c181bb9001f83db434c90087b67/scripts/run_lighthouse_tests.py#L109-L112

Reply:
These lines will be removed with the PR for https://github.com/oppia/oppia/issues/18260
We're temporarily supporting both Python and Docker while devs transfer to the Docker setup.